src/components/FieldWithValidation/FieldWithValidation.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import { Tooltip } from '../Tooltip';
+import { Text } from '../Text';
+import styles from './FieldWithValidation.scss';
+
+interface Props {
+    /** a single form field, e.g. Input, TextArea, etc. It should support context=”bad” prop */
+    children: React.ReactElement;
```

Comment:
I'll try to come up with more precise type for `children`, that reflects the idea that each clildren suppose to support context=bad bit later todayt

Reply:
if you can define an interface that allows any prop and also `context: 'bad' or something else` then you can use here `children: React.ReactElement<PropsThatSupportContext>`. I just wasn't sure how to write that type

Reply:
No success, let's postpone it